.eslintrc.js

```diff
@@ -20,38 +20,23 @@ module.exports = {
     browser: true,
   },
   rules: {
-    'arrow-parens': ['error', 'always'],
```

Comment:
Are all these non-Ember rules still necessary? If so, I can add them back. Didn't encounter any violations after removing them though.

Reply:
If linting is good without them, removal is good.